# Message describing the FSR sensors
Header header
float32 LFootFrontLeft
float32 LFootFrontRight
float32 LFootRearLeft
float32 LFootRearRight
geometry_msgs/Point32 LFootCenterOfPressure
float32 RFootFrontLeft
float32 RFootFrontRight
float32 RFootRearLeft
float32 RFootRearRight
geometry_msgs/Point32 RFootCenterOfPressure
